### DISCUSSION TITLE
Scrum Prep Milestone 3-1

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#179] [General]
Each team member will comment progress made through the past week.

**Comment by AlvinBadillo** [comment]
This week I had 3 exams and one presentation and was unable to make contributions to the development of the project.

**Reply by Cabinetman** [reply]
For this week I focused on balancing academic workload, as I had two major tests this week but i did Continue research and light planning for upcoming tasks in the project to stay aligned with our goals.

**Comment by franrov4** [comment]
Not much work done this week, time was most spent on other coursework.

**Comment by CovasJJ** [comment]
This week I wasn't able to do much work besides some research to prepare myself to start working on tasks.

**Comment by JadStelar** [comment]
Did not do much work for this class during the week. Will be opening future tasks during the day.

**Comment by cristian-barreras** [comment]
This week I had 2 exams and diligence to attend to, so I  was unable to work much on the project.

**Comment by PerezMarcelo** [comment]
Brainstorm ideas for tasks that will help us achieve our milestone goals.

**Comment by Christian2101** [comment]
Did not do much work for this class during the week.

**Comment by elliot-gomez** [comment]
This week, I focused on reviewing the lectures to brainstorm ways to incorporate the topics into the final stages of our project.
- Elliot J. Gomez Coriano

**Comment by JuanDelPueblo** [comment]
This week I was unable to do any progress on the project due to a heavy workload from other classes.

**Comment by gcal77** [comment]
This week I reviewed the milestone 3 goals and revised the course material for inspiration on future tasks.
